### PR TITLE
Jason/subscription 2 checkbox

### PR DESCRIPTION
**Pull request by duidae**
# Related Issue

[[會員制] 感興趣的議題頁](https://app.asana.com/0/1203699264568808/1203762836596954/f)

# Root Cause

# Solution

### CODE REVIEW

**Comment by Aylie-Chou** [comment]
Based on the design, the checkbox background color would need to be transparent when value is false.

// current code on storybook
螢幕截圖 2023-04-11 上午11 37 36

// design
螢幕截圖 2023-04-11 上午11 37 55

File: packages/react-components/src/checkbox/checkbox.stories.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  },
+}
+
+const Template = args => <Checkbox {...args} />
+export const checkbox = Template.bind({})
+checkbox.args = {
+  label: '藝術',
+  value: false,
+  disabled: false,
+  onChange: () => {},
```
Comment (Aylie-Chou) [inline packages/react-components/src/checkbox/checkbox.stories.js:21]: Could we add a story about how `onChange` works (ex. alert checked/uncheck ) ?

File: packages/react-components/src/button/utils/theme.js [hunk 2]
```diff
@@ -349,17 +349,17 @@ export const getActiveTextButtonTheme = theme => {
       }
     case TEXT_BUTTON_THEME.brand:
       return {
-        color: colorBrand.heavy,
+        color: colorBrand.dark,
```
Comment (Aylie-Chou) [inline packages/react-components/src/button/utils/theme.js:352]: question: why would we need to change color here ? (the changed color seems not match with design)

File: packages/react-components/src/checkbox/index.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+export const Checkbox = ({
+  value = false,
+  label = '',
+  disabled = false,
+  onChange = () => {},
+  ...props
+}) => {
+  const handleChange = e => {
+    onChange && onChange()
```
Comment (Aylie-Chou) [inline packages/react-components/src/checkbox/index.js:34]: may we pass event(`e`) in to onChange function

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]

File: packages/react-components/src/checkbox/index.js (hunk 3 above)
Comment (duidae) [reply packages/react-components/src/checkbox/index.js]: added

**Comment by duidae** [comment]
> Based on the design, the checkbox background color would need to be transparent when value is false.
> 
> // current code on storybook 螢幕截圖 2023-04-11 上午11 37 36
> 
> // design 螢幕截圖 2023-04-11 上午11 37 55

fixed, please have a look, thanks.

File: packages/react-components/src/button/utils/theme.js (hunk 2 above)
Comment (Aylie-Chou) [reply packages/react-components/src/button/utils/theme.js:352]: found it !

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**